README.md

```diff
@@ -8,8 +24,36 @@ $ npm install
 $ npm start
 ```
 
-Currently the site is compiled to static HTML, CSS, and JS where it can run on github pages.
+## Develop Workflow
+
+After successfully building and running the server, go to localhost:8080/
```

Comment:
it's usually good to make these actual links, just to make life one possible click easier.